src/extension.ts

```diff
@@ -9,6 +9,8 @@ import { initialize as initCommands } from "./commands";
 import { initialize as initRecommendations } from "./recommendation";
 import { initialize as initMisc, showReleaseNotesOnStart } from "./misc";
 import { showOverviewPageOnActivation, OverviewViewSerializer } from './overview';
+import { validateJavaRuntime } from "./java-runtime";
+import { valid } from 'semver';
```

Comment:
Is this member used anywhere?

Reply:
interesting...
The auto-completion put it here.